.github/workflows/ci.yml

```diff
@@ -35,6 +35,18 @@ jobs:
         uses: FreeRTOS/CI-CD-Github-Actions/coverage-cop@main
         with:
           path: ./build/coverage.info
+  default-config:
```

Comment:
(Nit) Suggestion for more readable naming from the perspective of seeing in the PR checks list
```suggestion
  build-with-default-config:
```